encord/objects/ontology_labels_impl.py

```diff
@@ -914,19 +914,30 @@ def data_type(self) -> DataType:
 
     @property
     def label_status(self) -> LabelStatus:
+        """
+        Returns current labeling status for the label row.
+
+        **Note**: is not supported for the workflow-based projects. See the documentation about workflows
```

Comment:
**Note**: This method is not supported for workflow-based projects. Please see our workflow documentation for more details.
 
 -> Is there a way to add a hyperlink here? If yes it would be good to link to the Workflow SDK section, if not then not :)